common/models/pages.py

```diff
@@ -313,6 +321,24 @@ class CategoryPage(MetadataPageMixin, Page):
         null=True,
         help_text='Detailed description of how we track the data for this particular category.'
     )
+
+    # Data Viz Configuration
+    viz_type = models.CharField(
+        max_length=255,
+        choices=CATEGORY_CHART_CHOICES,
+        default='none',
+        help_text='The type of chart shown in the category page. By default, no chart is shown.'
+    )
+    viz_data_start = models.PositiveIntegerField(
+        'Start Year',
+        blank=True,
+        null=True,
+    )
+    viz_data_end = models.PositiveIntegerField(
+        'End Year',
+        blank=True,
+        null=True,
+    )
```

Comment:
I think this is a fine start for how to structure visualization data on category pages, but I worry a little bit that it's not scalable if we ever wanted to have more than one chart on a page. Though maybe that's not something we're planning.

Reply:
Ok I just looked at the ticket and it says "multiple types of chart" which isn't necessarily the same as multiple charts, and which this data schema handles fine.

If we _did_ want to make it easier to add multiple charts in the future, I could actually see replacing these with a StreamField, where the different block types could correspond to the different chart types.